src/main/java/javapoz24/team3/rental/domain/rentact/RentActDTO.java

```diff
@@ -32,15 +34,19 @@ public static RentActDTO fromRentAct(RentAct rentAct) {
         if (optBooking.isPresent()) {
             referenceBookingId = optBooking.get().getId();
         }
+        Optional<Employee> optClosingEmployee = Optional.ofNullable(rentAct.getClosingEmployee());
+        Long closingEmployeeId = null;
+        if (optClosingEmployee.isPresent()) {
+            closingEmployeeId = optClosingEmployee.get().getId();
+        }
```

Comment:
Ja bym to tak zapisał:
Long closingEmployeeId = 
        Optional.ofNullable(rentAct.getClosingEmployee())
.map(optClosingEmployee -> optClosingEmployee.getId())
.orElseGet(() -> null);

Reply:
Zrobione. InteliJ sugeruje 
.map(BaseEntity::getId)
zamiast
map(optClosingEmployee -> optClosingEmployee.getId())

Niby działa ale budzi we mnie zdumienie...